react/client/src/App.tsx

```diff
@@ -7,7 +7,7 @@ import {
 } from 'react-router-dom';
 import { ThemeProvider, createMuiTheme } from '@material-ui/core/styles';
 
-import RoutingContextProvider from 'contexts/RoutingContext';
+import { RoutingContextProvider } from 'contexts/RoutingContext';
```

Comment:
Made a change so default was the context itself and secondary export was the provider